api/migrationtarget/client.go

```diff
@@ -127,6 +128,9 @@ func (c *Client) httpPost(modelUUID string, content io.ReadSeeker, endpoint, con
 	return nil
 }
 
+// OpenLogTransferStream connects to the migration logtransfer
+// endpoint on the target controller and returns a stream that JSON
+// logs records can be fed into. The objects written should be params.LogRecords.
```

Comment:
:) That was a method I added in an earlier PR, but I realised I hadn't documented it.